src/CbcParameters.hpp

```diff
@@ -464,7 +464,8 @@ class CBCLIB_EXPORT CbcParameters {
     PMBoundsAll,
     PMFixInt,
     PMFixAll,
-    PMEndMarker
+    PMEndMarker,
+    PMAllCsv
```

Comment:
Does the new `PMAllCsv` need to go before `PMEndMarker`? (@tkralphs)

Reply:
After reading [this](https://stackoverflow.com/questions/261963/how-can-i-iterate-over-an-enum), I think that is correct.
Pleas see my last commit.